en/docs/learn/configuring-sms-otp.md

```diff
@@ -64,22 +64,23 @@ provider.
 			`              $ctx.msg             ` respectively.
 
 		-   Optionally, enter the HTTP response code the SMS service
-			provider sends when the API is successfully called. Nexmo API
+			provider sends when the API is successfully called. Vonage API
 			and  Bulksms API sends 200 as the code, while Clickatell
 			and Plivo send 202. If this value is unknown, leave it blank and
 			the connector checks if the response is 200, 201 or 202.
 
-    ??? Note "Click here to configure Nexmo as the service provider."
+    ??? Note "Click here to configure Vonage as the service provider."
 
-		Follow the steps given below if Nexmo is used as the SMS provider:
+		"Nexmo" was rebranded to "Vonage", which is why some of the URLs and configurations below still contain the word "Nexmo".
```

Comment:
Can't we give the new URL that it's redirected to instead of giving this as a note since we've updated the other portions throughout the doc?

Reply:
They don't have new URLs for their dashboard yet and they haven't changed the SMS URL in their own docs either: https://developer.nexmo.com/api/sms